blocks/research/__link/research__link.css

```diff
@@ -0,0 +1,15 @@
+.research__link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: rgb(209, 41, 86);
+  font-size: 18px;
+  line-height: 30px;
+  text-decoration-line: underline;
```

Comment:
@Woolfs1423 Adding `text-decoration-line: underline;` for an `<a>` element is redundant because underlining is the default styling for links in most browsers. Browsers automatically apply underlining to hyperlinks to signify that they are clickable.

Removing this redundant style can simplify the code and avoid unnecessary overrides of default browser behavior.